src/wechaty_puppet/puppet.py

```diff
@@ -1,26 +1,78 @@
 """
-interface for puppet
+Python Wechaty - https://github.com/wechaty/python-wechaty
+
+Authors:    Huan LI (李卓桓) <https://github.com/huan>
+            Jingjing WU (吴京京) <https://github.com/wj-Mcat>
+
+2020-now @ Copyright Wechaty
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
 """
 from __future__ import annotations
 
-from abc import ABC
-from enum import Enum
-from .file_box import FileBox
+from typing import List, Optional
+
+from chatie_grpc.wechaty import (   # type: ignore
```

Comment:
I believe It would be better if we can create a separate file to import everything we need from the `chatie-grpc` module, then exported them for providing them to other modules because this can help us to keep a single file to community with the chatie-grpc.

e.g.

### schema.py

```py
from chatie-grpc import (
  bla...
  bla...
)

__all__ = [
  'bla...',
  'bla...',
]
```

### other.py

```py
from schema import (
  bla...,
  bla...,
)
```